src/resources/js/routes.js

```diff
@@ -11,5 +12,12 @@ export const router = new VueRouter({
         path: '/hello',
         name: 'Hello',
         component: Hello
-    }],
+    },
+    {
+        path: '/register',
```

Comment:
the path should be in french, as our app is in french, and that's something the user can see.

Reply:
no